.github/workflows/tests.yml

```diff
@@ -1,4 +1,4 @@
-name: Unit Tests
+name: Regression Tests
```

Comment:
Well, the Google tests are unit tests. Maybe "Unit and regression tests"? Then, you can add "regression" to the names of your three tests below.

Reply:
My thinking was that unit tests and simulator runs (functional tests) are both under the category of regression testing. Both types are run after every change to make sure nothing breaks.

Reply:
True, but regression testing involves comparison over time, while the unit tests are comparisons with something like "ground truth", such as ensuring that correct files and objects are created.